src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -313,13 +313,19 @@ internal void Update(double number)
 
                 case AggregationType.Histogram:
                     {
-                        int i;
-                        for (i = 0; i < this.histogramBuckets.ExplicitBounds.Length; i++)
+                        // If -inf is the first bound, +inf will be the second of two bounds
+                        // in which case there is only a single bucket, so set the bucket
+                        // index to increment should be zero.
+                        int i = 0;
+                        if (!double.IsNegativeInfinity(this.histogramBuckets.ExplicitBounds[0]))
```

Comment:
This is updating the hot path. Could you run the Histogram benchmarks for before and after this change?

Reply:
Updated benchmark file with results in the comments. It is faster than the existing comment but the benchmark has changed since I first branched from main. I'll retest this again when I'm back at my desktop PC